### PR TITLE
Update 1.1.1

### PR DESCRIPTION
**Pull request by expressmailing**
Corrections for some graphical and functionnal bugs

### CODE REVIEW

File: views/templates/admin/marketing_step0/marketing_step0.tpl [hunk 1]
```diff
@@ -15,51 +15,51 @@
         <img src="../modules/expressmailing/views/img/{l s='config_en.png' mod='expressmailing'}" border="0"><br>&nbsp;
     </div>
     <div class="panel" style="text-align: center">
-        <div style="display: inline-block; width:24%; min-width: 180px;">
-            <img src="../modules/expressmailing/views/img/email.png" border="0" style="display: inline-block">
-            <div style="display: inline-block; vertical-align: middle; padding-left: 1em; text-align: left">
+        <div style="display: inline-block; width:24%; min-width: 180px; text-align: left; vertical-align: top; margin-bottom: 1.5em">
+            <img src="../modules/expressmailing/views/img/email.png" border="0" style="display: inline-block; margin-bottom: .5em">
+            <div style="display: inline-block; vertical-align: top; padding-left: 1em; text-align: left; width: 18em">
```
Comment (Quetzacoalt91) [inline views/templates/admin/marketing_step0/marketing_step0.tpl]: You should think to move these CSS attributes to a CSS file.

File: views/templates/admin/marketing_step0/marketing_step0.tpl [hunk 2]
```diff
@@ -15,51 +15,51 @@
...
                     {/if}
                 </div>
             </div>
         </div>
-        <div style="display: inline-block; width:24%; min-width: 180px;">
-            <img src="../modules/expressmailing/views/img/sms.png" border="0" style="display: inline-block">
-            <div style="display: inline-block; vertical-align: middle; padding-left: 1em; text-align: left">
+        <div style="display: inline-block; width:24%; min-width: 180px; text-align: left; vertical-align: top; margin-bottom: 1.5em">
+            <img src="../modules/expressmailing/views/img/sms.png" border="0" style="display: inline-block; margin-bottom: .5em">
+            <div style="display: inline-block; vertical-align: top; padding-left: 1em; text-align: left; width: 18em">
```
Comment (Quetzacoalt91) [inline views/templates/admin/marketing_step0/marketing_step0.tpl]: As you can see, your CSS code is redundant. You can avoid this by moving it into a CSS file.

File: views/templates/admin/marketinge_step4/filters_free.tpl [hunk 3]
```diff
@@ -41,13 +40,13 @@
...
-				<div class="checkbox">
+				<div class="checkbox" style="float: none">
 					<label for="subscriptions_campaign_optin"><input type="checkbox" {if $subscriptions_campaign_optin}checked="checked"{/if} value="1" class="checkbox-inline" id="subscriptions_campaign_optin" name="subscriptions_campaign_optin">{l s='All Optin customers' mod='expressmailing'} &nbsp; <span class="badge" style="font-size: smaller">{$customers_suscriptions.total_optin|intval}</span></label>
 				</div>
-				<div class="checkbox">
+				<div class="checkbox" style="float: none">
 					<label for="subscriptions_campaign_newsletter"><input type="checkbox" {if $subscriptions_campaign_newsletter}checked="checked"{/if} value="1" class="checkbox-inline" id="subscriptions_campaign_newsletter" name="subscriptions_campaign_newsletter">{l s='All Newsletter suscribed customers' mod='expressmailing'} &nbsp; <span class="badge" style="font-size: smaller">{$customers_suscriptions.total_newsletter|intval}</span></label>
 				</div>
-				<div class="checkbox">
+				<div class="checkbox" style="float: none">
```
Comment (Quetzacoalt91) [inline views/templates/admin/marketinge_step4/filters_free.tpl]: Here again, we have multiple occurrences of ` style="float: none"`. You should move it in a CSS file.

**Comment by expressmailing** [comment]
Hi @Quetzacoalt91 
The inline css has been moved to the css file

Thanks

**Comment by Quetzacoalt91** [comment]
Hi @expressmailing,

Thanks, your PR has just been merged and the changes should be available from tomorrow.

Best regards

**Pull request merged**